pkg/fleetautoscalers/controller.go

```diff
@@ -152,6 +150,22 @@ func NewController(
 	return c
 }
 
+// NewExtensions binds the handlers to the webhook outside the initialization of the controller
+// initializes a new logger for extensions.
+func NewExtensions(wh *webhooks.WebHook) *Extensions {
+	ext := &Extensions{}
```

Comment:
nit: This and the next line can be combined:

```
ext := &Extensions{baseLogger: runtime.NewLoggerWithSource("FleetAutscalerExtensions")}
```

Reply:
Discussed offline, `NewExtensions` will match how `NewController` initializes the loggers.